src/vg_peer_service.erl

```diff
@@ -8,8 +8,16 @@
          stop/0,
          stop/1]).
 
-join(Node) when is_atom(Node) ->
-    partisan_peer_service:join(Node).
+join(Node) ->
+    {Name, Host, Port} = Node,
+    %% also need to handle fqdn here, but I think it's best fixed in partisan
+    Host1 = string:tokens(Host, "."),
+    HostIP0 = [list_to_integer(Octet) || Octet <- Host1],
+    HostIP = list_to_tuple(HostIP0),
+    %% this assumption of correlation is bad and we should fix; need a
+    %% better to do sys.config variations...
+    application:set_env(partisan, peer_port, Port + 10000),
```

Comment:
this is a silly hack.  Normally the peer port will be a well-known thing in the cluster.  I need to move the cluster testing to something with better isolation (docker compose, minikube or something), and then we can get rid of this.